PyInstaller/hooks/pre_find_module_path/hook-eyed3.plugins.py

```diff
@@ -0,0 +1,6 @@
+import eyed3.plugins
+from PyInstaller.utils.hooks import logger
+
+def pre_find_module_path(api):
+    api.search_dirs += eyed3.plugins.__path__
```

Comment:
`pre_find_module_path` is *rarely* used. Sure this is required?

Reply:
eyed3 imports thoses plugins like `import classic` but not `import eyed3.plugins.classic`. So I have to add the `plugins` directory to the python path.

Reply:
A `pre_find_module_path` hook only effects PyInstaller at freeze-time. But the hook already searches for the all sub-module in the correct directory. No need for this file.